client/dashboard/domains/domain-diagnostics/index.tsx

```diff
@@ -0,0 +1,198 @@
+import {
+	domainDnsEmailMutation,
+	domainDiagnosticsQuery,
+	domainNoticeMutation,
+} from '@automattic/api-queries';
+import { useMutation, useSuspenseQuery } from '@tanstack/react-query';
+import { useNavigate } from '@tanstack/react-router';
+import {
+	Button,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import Breadcrumbs from '../../app/breadcrumbs';
+import { domainOverviewRoute, domainRoute } from '../../app/router/domains';
+import InlineSupportLink from '../../components/inline-support-link';
+import Notice from '../../components/notice';
+import { PageHeader } from '../../components/page-header';
+import PageLayout from '../../components/page-layout';
+import { Text } from '../../components/text';
+
+export default function DomainDiagnostics() {
+	const { domainName } = domainRoute.useParams();
+	const navigate = useNavigate();
+
+	const { data: domainDiagnostics, refetch: refetchDomainDiagnostics } = useSuspenseQuery(
+		domainDiagnosticsQuery( domainName )
+	);
+
+	const { mutate: fixDnsIssues, isPending: isFixing } = useMutation( {
+		...domainDnsEmailMutation( domainName ),
+		meta: {
+			snackbar: {
+				success: __( 'The default email DNS records were successfully restored.' ),
```

Comment:
```suggestion
				success: __( 'Default email DNS records restored.' ),
```

Ref: https://github.com/Automattic/wp-calypso/blob/780603294d0ce3d8f56e241cc7f2116dd55e834a/client/dashboard/docs/typography-and-copy.md